src/index.js

```diff
@@ -101,8 +101,10 @@ function pingServer (address) {
 /**
  * Reads the arp table.
  */
-function arpAll () {
-  return cp.exec('arp -a', options).then(parseAll)
+function arpAll (_, skipNameResolution = false) {
+  const isWindows = process.platform.includes('win32')
```

Comment:
Does this also work on other platforms? If this only works on Windows we'd need to mention that in the README.

Reply:
This has no effect on windows machines as `arp -a` will be used. `arp -an` is not supported on this platform. However windows `arp` command doesn't seem to resolve host names either.

Reply:
Have modified the README to explain this